### PR TITLE
chore: issue template for new project checklists

### PR DESCRIPTION
**Pull request by rofe**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_project.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+---
+
+## Start a new Helix customer project
+
+### Set up the site
+
+- [ ] 1. Create a new GitHub repository, either under [Helix Sites](https://github.com/organizations/hlxsites/repositories/new) or the customer's own GitHub organization
+- [ ] 2. Create a content folder, either in the dedicated SharePoint site for [Helix Projects](https://adobe.sharepoint.com/sites/HelixProjects/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FHelixProjects%2FShared%20Documents%2Fsites)the customer's own Google Drive or SharePoint
+- [ ] 3. Share the folder with `helix@adobe.com` (Helix Bot) and copy the URL
```
Comment (tripodsan) [inline .github/ISSUE_TEMPLATE/new_project.md:15]: this is a bit tricky.... especially for sharepoint sites. it should not be the sharelink, but a relatively clean url, like:

https://adobe.sharepoint.com/sites/HelixProjects/Shared%20Documents/sites/westjet

File: .github/ISSUE_TEMPLATE/new_project.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+title: ''
+assignees: ''
+
+---
+
+## Start a new Helix customer project
+
+### Set up the site
+
+- [ ] 1. Create a new GitHub repository, either under [Helix Sites](https://github.com/organizations/hlxsites/repositories/new) or the customer's own GitHub organization
```
Comment (tripodsan) [inline .github/ISSUE_TEMPLATE/new_project.md:13]: mention th boilerplate?

File: .github/ISSUE_TEMPLATE/new_project.md (hunk 1 above)
Comment (rofe) [reply .github/ISSUE_TEMPLATE/new_project.md:15]: Ok, but how can we explain this so most people get it right the first time around? Or should we rather improve the system to be more resilient towards exotic sharepoint URLs?

File: .github/ISSUE_TEMPLATE/new_project.md (hunk 2 above)
Comment (rofe) [reply .github/ISSUE_TEMPLATE/new_project.md:13]: Of course 😅

File: .github/ISSUE_TEMPLATE/new_project.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- [ ] 9. Preview the document by opening the sidekick in Word Online or Google Docs and clicking _Preview_
+- [ ] 10. This will open a new browser tab with the project's homepage: `https://main--<project-name>--hlxsites.hlx.page/`
+
+**Congratulations!** 🎉
+
+### Set up communications
+
+#### Set up Slack (most SMBs)
+
+- [ ] 1. Create a new Slack channel `#helix-<project-name>` in the [Adobe Enterprise Support](https://adobeenterpri-izr7089.slack.com) instance
```
Comment (tripodsan) [inline .github/ISSUE_TEMPLATE/new_project.md:30]: url to be renamed....

File: .github/ISSUE_TEMPLATE/new_project.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- [ ] 2. Add the site's URL to the channel topic: `Preview https://main--<project-name>--hlxsites.hlx.page/`
+- [ ] 3. Type _Helix Bot_ (`@helix-bot`) _info_ and follow the its instructions to finish setting up the Slack channel for Helix
+- [ ] 4. Ping a Helix team member and ask them to send invites to the customer contacts' external email addresses
+
+#### Set up Teams (most Enterprise customers)
+
+- [ ] 1. Create a new Teams channel `#helix-<project-name>` in Adobe's Teams instance
```
Comment (tripodsan) [inline .github/ISSUE_TEMPLATE/new_project.md:37]: why not in our Adobe-Enterprise-Support instance?

File: .github/ISSUE_TEMPLATE/new_project.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+assignees: ''
+
+---
+
+## Start a new Helix customer project
+
+### Set up the site
+
+- [ ] 1. Create a new GitHub repository using the [adobe/helix-project-boilerplate](https://github.com/adobe/helix-project-boilerplate) repository as template, either under [Helix Sites](https://github.com/organizations/hlxsites/repositories/new) or the customer's own GitHub organization
+- [ ] 2. Create a content folder, either in the dedicated SharePoint site for [Helix Projects](https://adobe.sharepoint.com/sites/HelixProjects/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FHelixProjects%2FShared%20Documents%2Fsites) or in the customer's own Google Drive or SharePoint
```
Comment (tripodsan) [inline .github/ISSUE_TEMPLATE/new_project.md:14]: why not a new site in https://adobeenterprisesupportaem.sharepoint.com/sites ?

File: .github/ISSUE_TEMPLATE/new_project.md (hunk 1 above)
Comment (tripodsan) [reply .github/ISSUE_TEMPLATE/new_project.md]: we could either try to sanitize the urls when computing the contentbus id, or
reject the fstab if it's not clean and create a PR/issue to address the problem.

File: .github/ISSUE_TEMPLATE/new_project.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+- [ ] 3. Share the folder with the Helix Bot: `helix@adobe.com` (SharePoint) or `helix.integration@gmail.com` (Google Drive) and copy the URL
```
Comment (tripodsan) [inline .github/ISSUE_TEMPLATE/new_project.md:15]:
```suggestion
- [ ] 3a. Sharepoint: Give the Helix Bot: `helix@adobe.com` _edit_ permission to the folder and click on 'Copy Link' to retrieve the url.
- [ ] 3b. Google Drive: Share the folder with the Helix Bot: `helix.integration@gmail.com` and copy the URL of the folder from the browser's address bar.
```

**Pull request closed without merging**